src/styles/Comment.module.css

```
@@ -0,0 +1,175 @@
+.nextLink:hover {
+  text-decoration: underline;
+}
+
+.commentContent {
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  font-size: 9pt;
+  line-height: 1.5;
+  color: var(--color-primary);
+  margin-bottom: 5px;
+  word-wrap: break-word;
+  word-break: break-word;
```

Stylelint (17.6.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)